examples/graphs/node2vec_movielens.py

```diff
@@ -0,0 +1,582 @@
+"""
+Title: Graph representation learning with node2vec.
+Author: [Khalid Salama](https://www.linkedin.com/in/khalid-salama-24403144/)
+Date created: 2021/05/15
+Last modified: 2021/05/15
+Description: Implementing the node2vec to generate embeddings for movies.
+"""
+
+"""
+## Introduction
+
+
+Learning useful representations from objects structured as graphs is useful for
+a variety of ML applications, and leads to greater predictive power.
+[Graph Representation Learning](https://www.cs.mcgill.ca/~wlh/grl_book/) aims to
+learn embeddings for the graph nodes, which can be used for a variety of ML tasks
+such as node label prediction and link prediction.
```

Comment:
"such as node label prediction and link prediction" -> give concrete examples

Reply:
Done